content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst

```diff
@@ -2,60 +2,114 @@
+Configuration --> Locations`. Doing so opens a list of all existing locations. Click :guilabel:`New`
+to open a new location creation form.
+
+In the :guilabel:`Location Name` field, assign a name for the new location. In the :guilabel:`Parent
+Location` field, assign the location that this new location exists in.
+
+.. example::
+   A new location `Gate 1` is created. Its parent location is `Dispatch Zone`. In the
+   :guilabel:`Parent Location` for `Gate 1`, select `Dispatch Zone`.
+
+In the :guilabel:`Additional Information` section, fill out the necessary details for the new
+location.
+
+In the :guilabel:`Location Type` field, select which type of location this is. There are seven
+different types of locations to choose from:
 
-- :guilabel:`Location Name`: the name that will be used to reference the location
-- :guilabel:`Parent Location`: the location or warehouse that the new location exists within
-- :guilabel:`Location Type`: choose the category that the location belongs to
-- :guilabel:`Company`: the company that owns the warehouse that the location is inside of
+- :guilabel:`Vendor Location`: Virtual location representing the source location for products coming
+  from vendors.
+- :guilabel:`View`: Virtual location used to create a hierarchical structure for a warehouse;
+  aggregates its child locations. **Cannot directly contain products**.
+- :guilabel:`Internal Location`: Physical location inside a warehouse.
+- :guilabel:`Customer Location`: Virtual location representing the destination location for products
+  sent to customers.
+- :guilabel:`Inventory Loss`: Virtual location serving as counterpart for inventory operations. Used
+  to correct stock levels of physical inventory.
+- :guilabel:`Production`: Virtual counterpart location for production operations. This location
+  consumes components and produces finished products.
+- :guilabel:`Transit Location`: Counterpart location that should be used in inter-company or
+  inter-warehouse operations.
+
+.. image:: use_locations/use-locations-new-location-form.png
+   :align: center
+   :alt: Additional Information section of new location creation form.
+
+Configure the remaining fields in the :guilabel:`Additional Information` field as follows:
+
+- :guilabel:`Company`: the company whose warehouse the location is inside of. Leave this field blank
+  if this location is shared between companies.
 - :guilabel:`Is a Scrap Location?`: check this box to allow for scrapped/damaged goods to be stored
-  in this location
+  in this location.
 - :guilabel:`Is a Return Location?`: check this box to allow products to be returned to this
-  location
-- :guilabel:`Barcode`: the barcode number assigned to the location
-- :guilabel:`Removal Strategy`: the :doc:`strategy <../removal_strategies>` for how items
-  should be taken from inventory
+  location.
+- :guilabel:`Barcode`: the barcode number assigned to the location.
```

Comment:
location barcodes can also be letters (:
```suggestion
- :guilabel:`Barcode`: the barcode assigned to the location.
```

Reply:
Nice catch - Suggestion implemented ✅